packages/shared-fixtures/fixtures/typescript/basics/never-type-param.src.ts

```diff
@@ -1,2 +1,2 @@
-const x: X<never>;
+var x: X<never>;
```

Comment:
For my learning: why the change?

Reply:
babel reports now errors for unidealized const variables, witch is tested in

```
    /**
     * [BABEL ERRORED, BUT TS-ESTREE DID NOT]
     * SyntaxError: Missing initializer in const declaration.
     */
    'var-with-definite-assignment',
```